entrypoint.sh

```diff
@@ -33,4 +33,3 @@ else
    wait_for_db
```

Comment:
Because Github, I can't write a comment up on L18....

We'll actually want to change L18 to simply be `/sledger`.  This will enable the new, environment parameter based functionality you are implementing below in `sledger.go`.